{"timestamp":"2025-08-11T00:19:50.516023","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-08-11T00:19:50.523379","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/process_kindle_reviews_dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-08-11T00:19:52.193679","level":"error","event":"Failed to establish connection to Docker host unix://var/run/docker.sock: Error while fetching server API version: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory'))","logger":"airflow.task.hooks.airflow.providers.docker.hooks.docker.DockerHook"}
{"timestamp":"2025-08-11T00:19:52.194780","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"AirflowException","exc_value":"Failed to establish connection to any given Docker hosts.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":838,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1130,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py","lineno":476,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py","lineno":349,"name":"cli"},{"filename":"/usr/local/lib/python3.12/functools.py","lineno":998,"name":"__get__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/hooks/docker.py","lineno":161,"name":"api_client"}]}]}
